productivity-suite/piercing-library/src/message-bus/message-bus-prop.ts

```diff
@@ -0,0 +1 @@
+export const messageBusProp = Symbol.for("fragment-message-bus");
```

Comment:
no, it's because of the following code:
![Screenshot 2022-11-04 at 15 46 17](https://user-images.githubusercontent.com/61631103/200017982-88f470a7-5f32-414f-a42f-ecd76935e2b4.png)

That code is in the host web component which gets converted into an inline script, so in order not to slow down thing we need to try to make the inline script as small as possible, if i had:
`import { messageBusProp } from '../message-bus/message-bus.ts'` all the content of the message-bus file would also be added to the inline script (which is not that much but I was trying to keep the inline script as small as possible). So by splitting things in multiple files we can get manage to get into the inline script only what's really really necessary.

I hope it makes sense, let me know if you see some issues with that 🙂